lib/navigation/description.rb

```diff
@@ -0,0 +1,14 @@
+class Navigation::Description < Navigation
+  def initialize(page)
+    @page = page
+  end
+  def to_s
+    if @page.description
+      description = @page.description
+    elsif @page.parent&.description
+      description = @page.parent.description
+    else
+      description = Entities::Pages.all[0].description
+    end
```

Comment:
to make this clearer, I would move this to a method called `text`. Then in `to_s` call the `text` method. I don't think you need the `description =` at the beggining of each if statement. 

I don't think these changes should break the tests.